### ISSUE TITLE
Sound in Safari

### ISSUE THREAD

**Issue by rezwits** [mtgred/netrunner#2392]
Sound works in Safari Technology Preview Release 24 (Safari 10.2, WebKit 12604.1.6).
but not in Current Safari Version 10.0.3 (12602.4.8).

Not everyone has the Preview... just saying...

**Comment by Saintis** [comment]
I have Safari "Version 10.0.3 (11602.4.8.0.1)". Sound works for me (at least when I played yesterday).

**Comment by rezwits** [comment]
Yeah, it’s been intermittent for me. Like when I first went in, it was working fine, and now nothing.  And I was like I know the sound works, so I fired up Preview, and it’s been flawless so far.  I don’t think I changed any “Safari Setting” to .turn off the sound for the site.

I have just decide to use preview all the time now anyway… oh well…

Thomas Aldershof II
(rezwits@me.com <mailto:rezwits@me.com>)
